css/print.scss

```
@@ -0,0 +1,83 @@
+@media print {
+	@page {
+		size: A4;
+		margin: 2.5cm 2cm 2cm 2.5cm; 
+	}
+
+	// Hide sidebar if open
+	#app-sidebar-vue {
+		display: none;
+	}
+
+	// Hide Content behind modal
+	#content {
+		display: none;
+	}
+
+	#viewer {
```

We should avoid global selector like that.

```suggestion
	#viewer[data-handler="text"] {
```

---

Fixed that